otland/forgottenserver
# feat: add workflow to format Lua files in pull requests

**Description**
This pull request adds a workflow to the project that automatically formats Lua files in pull requests. The goal is to ensure that all Lua files are formatted according to established guidelines, promoting consistency and improving code readability.

**What was done**
- Added a formatting workflow for Lua files.
- Configured to run automatically on pull requests.
- The workflow uses [formatting tool] to apply the standard formatting.

**How to test**
- Create a pull request with Lua files that are not correctly formatted.
- Verify that the formatting workflow runs automatically.
- Ensure that Lua files are formatted according to the established guidelines.

**Notes**
- Ensure that the formatting tool is correctly configured in your local environment to avoid discrepancies.
- Formatting will be applied automatically to code before merging into the main branch.
